client/db/bolt/upgrades_test.go

```diff
@@ -0,0 +1,98 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package bolt
+
+import (
+	"compress/gzip"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"decred.org/dcrdex/dex/encode"
+	"go.etcd.io/bbolt"
+)
+
+var dbUpgradeTests = [...]struct {
+	verify   func(*testing.T, *bbolt.DB)
+	filename string // in testdata directory
+}{
+	{verifyV1Upgrade, "v0.db.gz"},
+}
+
+func TestUpgrades(t *testing.T) {
+	t.Parallel()
+
+	d, err := ioutil.TempDir("", "dcrdex_test_upgrades")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Run("group", func(t *testing.T) {
```

Comment:
Why use `Run` here? What does `"group"` mean?

Reply:
Just grouping the upgrade tests here and running them in parallel, "group" is the name of the subtest.

Reply:
This outside `t.Run` does nothing. You already are running the upgrade tests in parallel, since you are doing a `t.Run` with `t.Parallel` for each `dbUpgradeTests`. There's only one "main subtest", so there's no need for it.